matter/efr32/efr32mg24/BRD4186C/autogen/sl_event_handler.c

```diff
@@ -91,6 +99,9 @@ void sl_service_init(void)
   sl_sleeptimer_init();
   sl_hfxo_manager_init();
   sl_mbedtls_init();
+#ifdef CHIP_917
+  sl_mpu_disable_execute_from_ram();
```

Comment:
mpu is not enable in smg/csa. I  doubt you need this

Reply:
Removed